src/continuous-validation/body-pair-collision.cc

```diff
@@ -143,8 +143,8 @@ namespace hpp {
       }
 
       bool BodyPairCollision::computeDistanceLowerBound(value_type &distanceLowerBound,
-        CollisionValidationReportPtr_t& report,
-        pinocchio::DeviceData& data) const
+        ValidationReportPtr_t& report,
+        const pinocchio::DeviceData& data) const
```

Comment:
Any reason to make this const ?
The lines below trigger a compilation error. Does this compile for you ?
```cpp
          pinocchio::FclConstCollisionObjectPtr_t object_a = _pair->first ->fcl (data);
          pinocchio::FclConstCollisionObjectPtr_t object_b = _pair->second->fcl (data);
```

Reply:
Forget this. I wasn't up-to-date in hpp-pinocchio.